Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -362,28 +400,24 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
         this.dispatchEventToListeners(WI.InlineSwatch.Event.Activated);
 
         switch (this._type) {
-        case WI.InlineSwatch.Type.CubicBezierTimingFunction:
```

Comment:
i think what you did in a previous commit was to have `WI.ColorPicker` set `this.color = WI.Color.fromString("white");` in order to initialize the component `<input>` to have the right starting `height`.  i think that approach is better than this (and yes i realize that this is what i'd suggested offline)